modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/DynamicInlineScroll.es.js

```diff
@@ -150,16 +151,18 @@ class DynamicInlineScroll extends PortletBase {
 	 */
 	onScroll_(event) {
 		const {cur, initialPages, pages} = this;
-		const {target} = event;
+		const {delegateTarget} = event;
```

Comment:
This one looks slightly suspicious.. why was it previously using `target` but now needs `delegateTarget`?

Reply:
I agree, I went on autopilot there a little bit, it makes sense that it's `delegateTarget` because it looks for very specific attributes that most elements won't have, so I _assumed_ that `target` could be lots of elements, whereas `delegateTarget` will be the one specified by the `selector`.

I'll try and test it manually to make sure.